.github/workflows/github_pr.yml

```
@@ -22,8 +22,7 @@ jobs:
         os: [ ubuntu-18.04, macOS-10.15, windows-2019 ]
         dc: [ dmd-2.091.0, ldc-1.20.1, ldc-1.19.0 ]
         exclude:
-          # Not supported yet
-          - os: windows-2019
+          - { os: windows-2019, dc: dmd-2.091.0 }
```

```suggestion
          - ubuntu-18.04
          - macOS-10.15
          - { os: windows-2019, dc: dmd-2.091.0 }
```

For the time being, to allow you to get Windows working